airflow/models/taskinstance.py

```diff
@@ -2306,12 +2306,14 @@ def emit_state_change_metric(self, new_state: TaskInstanceState) -> None:
         elif new_state == TaskInstanceState.QUEUED:
             metric_name = "scheduled_duration"
             if self.start_date is None:
-                # same comment as above
```

Comment:
I have a question on the state change timing functionality. In multi scheduler mode, Does the state change timing  captured?

Reply:
Not sure I understand the question. Do you mean to ask if this metric work with multiple schedulers? It should since each task should only be handled by one scheduler, so the schedulers should not step on each other on this.